hassio/src/addon-view/hassio-addon-config.ts

```diff
@@ -0,0 +1,163 @@
+import "@polymer/iron-autogrow-textarea/iron-autogrow-textarea";
+import "@material/mwc-button";
+import "@polymer/paper-card/paper-card";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+
+import { HomeAssistant } from "../../../src/types";
+import {
+  HassioAddonDetails,
+  setHassioAddonOption,
+  HassioAddonSetOptionParams,
+} from "../../../src/data/hassio";
+import { hassioStyle } from "../resources/hassio-style";
+import { haStyle } from "../../../src/resources/styles";
+import { PolymerChangedEvent } from "../../../src/polymer-types";
+import { fireEvent } from "../../../src/common/dom/fire_event";
+
+@customElement("hassio-addon-config")
+class HassioAddonConfig extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public addon!: HassioAddonDetails;
+  @property() protected error?: string;
+  @property() private _config!: string;
+  @property({ type: Boolean }) private _configHasChanged = false;
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this._config = JSON.stringify(this.addon.options, null, 2);
```

Comment:
I don't think we want to do this on `connected` but when the addon changes? So in updated?

Reply:
You are right.